### PR TITLE
feat: [SRM-10611]: Add Tags to Execution logs and External API call Logs

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/70-pipeline/components/LogsContent/LogsState/types.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export interface SearchData {
 /**
  * Keys in the json from server
  */
-export type TextKeys = 'level' | 'out' | 'time'
+export type TextKeys = 'level' | 'out' | 'time' | 'tags'
```
Comment (praneshg239) [inline src/modules/70-pipeline/components/LogsContent/LogsState/types.ts:53]: If these are the keys from server, shall we take these from `services/index.ts` file?

Comment (praneshg239) [reply src/modules/70-pipeline/components/LogsContent/LogsState/types.ts:53]: So that it will in sync with backend

**Review by praneshg239 — APPROVED** [review]

**Comment by praneshg239** [comment]
retrigger jest

File: src/modules/70-pipeline/components/LogsContent/components/MultiLogLine/MultiLogLine.tsx [hunk 2]
```diff
@@ -142,6 +142,26 @@ export function MultiLogLine(props: MultiLogLineProps): React.ReactElement {
...
+          searchText,
+          searchIndices: searchIndices?.level,
+          currentSearchIndex
+        })}
+      />
+      <LogLine
+        skipLinkify
+        className={css.level}
+        data={getTextWithSearchMarkers({
+          txt: text.tags,
```
Comment (vkbansal) [inline src/modules/70-pipeline/components/LogsContent/components/MultiLogLine/MultiLogLine.tsx:159]: Why not prepend `Tags` text here?

**Comment by kapilkumawat86** [comment]
please add screenshots for the change

**Comment by vasuki-kalluri** [comment]
image

**Comment by vasuki-kalluri** [comment]
image

File: src/modules/70-pipeline/components/LogsContent/components/MultiLogLine/MultiLogLine.tsx (hunk 2 above)
Comment (vasuki-kalluri) [reply src/modules/70-pipeline/components/LogsContent/components/MultiLogLine/MultiLogLine.tsx:159]: Search on Tags also needs to work.

**Comment by vasuki-kalluri** [comment]
retrigger jest

**Comment by vasuki-kalluri** [comment]
retrigger jest

**Comment by vasuki-kalluri** [comment]
retrigger coverage

**Comment by vasuki-kalluri** [comment]
retrigger jest

**Review by praneshg239 — APPROVED** [review]

**Review by sahithikolichala — APPROVED** [review]

**Review by kapilkumawat86 — COMMENTED** [review]
@vivekharness please review

**Review by kapilkumawat86 — APPROVED** [review]

**Review by vkbansal — APPROVED** [review]

**Pull request merged**